test/factories/products.rb

```diff
@@ -0,0 +1,12 @@
+FactoryBot.define do
+  factory :product do
+    title { "MyString" }
+    description { "MyText" }
+    status { "MyString" }
+    category { "MyString" }
+    price_cents { 1 }
+    price_currency { "MyString" }
+    created_at { "2024-05-23 10:34:41" }
+    updated_at { "2024-05-23 10:34:41" }
```

Comment:
Same. let's remove it if we won't use it.

Reply:
ok